engine/src/main/java/org/terasology/engine/rendering/dag/RenderGraph.java

```diff
@@ -529,13 +533,14 @@ public void reconnectInputToOutput(String fromNodeUri, int outputId, Node toNode
      */
     private void reconnectInputToOutput(Node toNode, int inputId, DependencyConnection fromConnection,
                                         ConnectionType connectionType, boolean disconnectPrevious) {
-        logger.debug("Attempting reconnection of {} to {}'s output.", toNode.getUri(), fromConnection.getParentNode());
+        logger.atDebug().addArgument(() -> toNode.getUri()).addArgument(() -> fromConnection.getParentNode()).
+                log("Attempting reconnection of {} to {}'s output.");
```

Comment:
simple getter methods
```suggestion
        logger.debug("Attempting reconnection of {} to {}'s output.", toNode.getUri(), fromConnection.getParentNode()); //NOPMD
```